src/backend/rest.ts

```diff
@@ -72,10 +72,19 @@ export default class RestApiBackend implements Backend {
       url: '/version',
       method: 'GET',
     });
-    // TODO: check if this version matches the edition!
+    const checkMkVersion: string = result.data.versions.checkmk;
+    if (checkMkVersion.startsWith("2.0") || checkMkVersion.startsWith("1.6")) {
+      throw new Error(`CheckMK version ${checkMkVersion} is not supported for this plugin, but you can set the backend to the '< 2.2' version and use at your own risk.`)
+    }
+    if (checkMkVersion.startsWith("2.1")) {
+      throw new Error(`CheckMK version ${checkMkVersion} does not support querying graph data using this backend. Please set the backend option to '< 2.2' and make sure to have the Web Api enabled.`)
```

Comment:
can you try to reword this without using "backend"? this might be confusing for the customer as they simply specify the version and don't know about the different backends?

For example:

> Version mismatch! Version ">= 2.2" is selected in data source settings, but version 2.1 was detected.